### PR TITLE
[WIP] switch to registry.redhat.io for infra images

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: roles/ansible_service_broker/vars/openshift-enterprise.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 __ansible_service_broker_registry_type: rhcc
 __ansible_service_broker_registry_name: rh
-__ansible_service_broker_registry_url: "https://registry.access.redhat.com"
+__ansible_service_broker_registry_url: "https://registry.redhat.io"
```
Comment (bparees) [inline roles/ansible_service_broker/vars/openshift-enterprise.yml]: @jwmatthews if this is what i think it is, i'm guessing it's going to be broken by this change (registry.redhat.io requires auth) so we're going to need to make it clear that the registry_user and registry_password fields are not optional.

**Comment by bparees** [comment]
[(5) Tolerate auth-enabled redhat registry [builds]](https://trello.com/c/Howwzs2c/1492-5-tolerate-auth-enabled-redhat-registry-builds)

File: roles/openshift_storage_glusterfs/defaults/main.yml [hunk 2]
```diff
@@ -5,22 +5,22 @@
 # becomes "oreg.example.com/rhgs3/rhgs-server-rhel7:latest"
 l_gluster_image_dict:
   origin: "docker.io/gluster/gluster-centos:latest"
-  openshift-enterprise: "registry.access.redhat.com/rhgs3/rhgs-server-rhel7:latest"
+  openshift-enterprise: "registry.redhat.io/rhgs3/rhgs-server-rhel7:latest"
```
Comment (bparees) [inline roles/openshift_storage_glusterfs/defaults/main.yml]: @sdodson this is another case that needs to be examined... will these values get replaced from oreg_url?

Comment (michaelgugino) [reply roles/openshift_storage_glusterfs/defaults/main.yml]: These are defaults.  They will be affected if someone sets oreg_url.

Comment (bparees) [reply roles/openshift_storage_glusterfs/defaults/main.yml]: cool

**Comment by bparees** [comment]
@sdodson @michaelgugino removed the WIP, i think this is ready for review and potential merge (not that it's going to break everyone not setting "oreg_url" since they're going to try to get images from registry.redhat.io and the images aren't there.  But 3.11 images also aren't on registry.access.redhat.com, so i assume everyone trying to use the 3.11 install is setting oreg_url anyway.

**Comment by michaelgugino** [comment]
@bparees most of our CI runs on origin here, so we should be okay for merging purposes.

**Pull request closed without merging**